README.rst

```diff
@@ -12,6 +12,23 @@ powered by the BigQuery engine.
 BigQuery DataFrames is an open-source package. You can run
 ``pip install --upgrade bigframes`` to install the latest version.
 
+.. raw:: html
+
+    <div style="border: 1px solid #f5c6cb;
+    background-color: #f8d7da; color: #721c24; padding: 15px; margin-bottom: 20px; border-radius: 4px; font-size: 90%;">
+        <p style="margin-top: 0; font-weight: bold;">⚠️ Breaking Changes in BigQuery DataFrames v2.0</p>
+        <p>Version 2.0 introduces breaking changes for improved security and performance. Key default behaviors have changed:</p>
+        <ul>
+            <li><strong>Large Results (&gt;10GB):</strong> The default value for <code>allow_large_results</code> has changed to False. Methods like <code>to_pandas()</code> will now fail if the query result's compressed data size exceeds 10GB, unless large results are explicitly permitted.</li>
+            <li><strong>Remote Function Security:</strong>The library no longer automatically lets the Compute Engine default service account become the identity of the Cloud Run functions. If that is desired, it has to be indicated by passing cloud_function_service_account="default". And network ingress now defaults to "internal-only".</li>
+            <li><strong>@remote_function Argument Passing:</strong> Arguments to remote_function must now be passed using keyword syntax, as positional arguments are no longer supported.</li>
```

Comment:
I have https://github.com/googleapis/python-bigquery-dataframes/pull/1560 open to partially revert this change.

```suggestion
            <li><strong>@remote_function Argument Passing:</strong> Arguments other than input_types, output_type, and dataset to remote_function must now be passed using keyword syntax, as positional arguments are no longer supported.</li>
```